src/Command/RecipesCommand.php

```diff
@@ -165,6 +165,10 @@ private function displayPackageInformation(Recipe $recipe)
         $lockBranch = $recipeLock['recipe']['branch'] ?? null;
         $lockVersion = $recipeLock['recipe']['version'] ?? $recipeLock['version'] ?? null;
 
+        if ('master' === $lockBranch && 'github.com/symfony/recipes' === $lockRepo) {
```

Comment:
We should also do it for the recipes-contrib :
```suggestion
        if ('master' === $lockBranch && \in_array($lockRepo, ['github.com/symfony/recipes', 'github.com/symfony/recipes-contrib'])) {
```